src/components/input-field/input-field.tsx

```
@@ -321,21 +321,26 @@ export class InputField {
+                hasValue={!!this.value}
+                hasLeadingIcon={!!this.leadingIcon}
+            >
+                <label slot="content" class={this.getContainerClassList()}>
+                    {this.renderLeadingIcon()}
+                    {this.renderPrefix()}
+                    {this.renderFormattedNumber()}
+                    {this.renderInput(properties)}
+                    {this.renderSuffix()}
+                    {this.renderTextarea(properties)}
+                    {this.renderTrailingLinkOrButton()}
+                </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)